features/facebookImport/src/popUps/baseInfoPopUp.css

```diff
@@ -0,0 +1,19 @@
+.base-info-contents {
```

Comment:
Yes, it has been moved. But I didn't use `git mv` 😟

Reply:
No problem, it's no  biggie. Helps with revision, though, because if there was no change there would have been nothing to review, only if there's an additional diff.